src/GameLogic/Entities/Fireball.ts

```diff
@@ -51,6 +52,11 @@ export class Fireball implements Entity {
 		return clone;
 	}
 
+	public getNextMoveDetails(): EntityMovement {
+		// Do nothing
+		return new EntityMovement(this, -1, -1);
```

Comment:
Shouldn't it return its actual position, even if the proper logic is not really here yet?